xarray/coding/times.py

```diff
@@ -291,7 +279,12 @@ def cftime_to_nptime(times):
     times = np.asarray(times)
     new = np.empty(times.shape, dtype='M8[ns]')
     for i, t in np.ndenumerate(times):
-        dt = datetime(t.year, t.month, t.day, t.hour, t.minute, t.second)
```

Comment:
Was there a reason we did not include the `microsecond` attribute of the datetime here previously?